Pin the redis package to version 3.5.3 (the last version that supports both
Python 2 and 3).

Signed-off-by: Saikrishna Arcot <sarcot@microsoft.com>

diff --git a/requirements.txt b/requirements.txt
index 7800f0f..3fc0632 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1 +1 @@
-redis
+redis==3.5.3
diff --git a/setup.py b/setup.py
index 8ccf31f..6db9ec4 100644
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,7 @@ setup(name=package_name,
     author_email='oleg@bsdpower.com',
     url='http://github.com/p/redis-dump-load',
     py_modules=['redisdl'],
-    install_requires=['redis'],
+    install_requires=['redis==3.5.3'],
     data_files=[
         (doc_dir, data_files),
     ],
